data/polityka/polityczne.csv

```diff
@@ -0,0 +1 @@
+rząd,wybory,partia_polityczna,prezydent,premier,posłowie,senat,sejm,polityka_zagraniczna,polityka_społeczna,polityka_gospodarcza,konstytucja,ustroj,demokracja,autorytaryzm,prawo,lobbying,dyplomacja,prawo_wyborcze,kampania_wyborcza,opozycja,koalicja,ustroj_parlamentarny,ustroj_prezydencki,konflikt_polityczny,polityka_publiczna,administracja_państwowa,biurokracja,deklaracja_praw_człowieka,prawo_konstytucyjne,system_polityczny,ład_polityczny,konsulat,ambasada,diplomacja_publiczna,rozprawa_polityczna,kodeks_wyborczy,referendum,projekt_ustawy,izba,parlament_europejski,sąd_konstytucyjny,sąd_administracyjny,trybunał,system_jednoizbowy,system_dwuizbowy,polityka_fiskalna,prawo_administracyjne,polityka_bezpieczeństwa,partycypacja_obywatelska
```

Comment:
Shouldn't the underscores (`_`) be spaces instead? Or shouldn't the code account for those?